src/main/groovy/com/github/dispader/manifesto/ManifestoPlugin.groovy

```diff
@@ -10,67 +10,23 @@ class ManifestoPlugin implements Plugin<Project> {
 
         project.extensions.create('manifesto', ManifestoPluginExtension, project)
 
+        def version = new Version()
+        project.version = version.version
```

Comment:
This change wigs me out a little bit... it looks like it gets run only at project configuration time (not after the project's been evaluated, like the stuff in the`whenPluginAdded` code below.  (`whenPluginAdded` is no way to execute that code by the way— it's just awful, and should be in in some kind of `afterEvaluate` block: I just couldn't get that to work because I'm really awful at writing Gradle plugins, apparently.)  Point for this comment is that I really wonder if the version will get updated when folks perform builds with something like the Gradle Daemon running— I dunno that they won't, but it seems like we're getting version information when this plugin is added, rather than when the `Jar` tasks are evaluated.